framework/src/ble_manager/ble_manager_state.c

```diff
@@ -925,10 +943,14 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 			}
 		}
 
-		if (ctx && ctx->callbacks.ble_client_device_connected_cb) {
-			ctx->callbacks.ble_client_device_connected_cb((ble_client_ctx *)ctx, data);
+		if (ctx && ctx->callbacks.connected_cb) {
+			blemgr_msg_params param = {
+				.count = 3,
+				.param = {ctx->callbacks.connected_cb, ctx, msg->param},
+				.evt = msg->event
+			};
+			ble_queue_enque(BLE_QUEUE_EVT_PRI_HIGH, &param);
```

Comment:
It is almost duplicated code, you can reduce it.

Reply:
Okay, I will remove duplicated code